qiskit/circuit/library/standard_gates/u3.py

```diff
@@ -24,6 +24,18 @@
 class U3Gate(Gate):
     r"""Generic single-qubit rotation gate with 3 Euler angles.
 
+    .. warning::
+
+       This gate is deprecated. Instead, the following replacements should be used
+
+       .. math::
+
+           U3(\theta, \phi, \lambda) =  U(\theta, \phi, \lambda)
+       .. codeblock:: python
```

Comment:
Does this need a linebreak?
```suggestion
           U3(\theta, \phi, \lambda) =  U(\theta, \phi, \lambda)

       .. codeblock:: python
```